src/pages/strategy/tabs/expcalc/expcalc.js

```diff
@@ -227,10 +227,12 @@
 			
 			// Exp Modifier: MVP
 			$(".ship_mvp .ship_value", goalBox).text( grindData[6]?"Yes":"No" );
+			$(".ship_mvp .ship_value", goalBox).css('color',(grindData[6]? "#e33" : "#039")); //yes : light red, no : deep blue
 			if(grindData[6]===1){ expPerSortie = expPerSortie * 2; }
 			
 			// Exp Modifier: FLAGSHIP
 			$(".ship_fs .ship_value", goalBox).text( grindData[5]?"Yes":"No" );
+			$(".ship_mvp .ship_value", goalBox).css('color',(grindData[6]? "#e33" : "#039")); 
```

Comment:
wow. so notice slow.... many obvious...
 can someone on master branch fix this for me XD
*screwed up first brach*